dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Register/DateOfBirthPage.cshtml.cs

```diff
@@ -37,12 +42,23 @@ public async Task<IActionResult> OnPost()
         var authenticationState = HttpContext.GetAuthenticationState();
         authenticationState.OnDateOfBirthSet((DateOnly)DateOfBirth!);
 
+        var users = await _userSearchService.FindUsers(
+            authenticationState.FirstName!,
+            authenticationState.LastName!,
+            (DateOnly)DateOfBirth!);
+
+        if (users.Length > 0)
+        {
+            authenticationState.OnExistingAccountFound(users[0]);
```

Comment:
Unclear what we do if we match on more than one user

Reply:
Created a new card for the designers to figure this one out.